contracts/WstETHPriceFeed.sol

```diff
@@ -70,7 +70,7 @@ contract WstETHPriceFeed is AggregatorV3Interface {
     ) {
         (uint80 roundId, int256 stETHPrice, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound) = AggregatorV3Interface(stETHtoUSDPriceFeed).latestRoundData();
         uint256 tokensPerStEth = IWstETH(wstETH).tokensPerStEth();
-        int price = stETHPrice * signed256(wstETHScale) / signed256(tokensPerStEth);
+        int256 price = stETHPrice * int256(wstETHScale) / signed256(tokensPerStEth);
```

Comment:
Since we are assuming `wstETHScale` is already guaranteed to be < int256.max?

Reply:
yeah, that's the assumption:

https://github.com/compound-finance/comet/pull/587#discussion_r1009744397

Reply:
Yep 👍 comet has `10**18` max constant and changing that would be a massive breaking change